### PR TITLE
fix bug in mt/st templates for having separate token lists

### PR DESCRIPTION
**Pull request by brianyan918**

### CODE REVIEW

**Comment by siddalmia** [comment]
Looks good! Good catch.

**Comment by brianyan918** [comment]
I added a corresponding change to st/espnet_model.py. These bug fixes are tested and I think that now separate src and target vocabs is fully supported.

**Comment by siddalmia** [comment]
@brianyan918 Good catch of this subtle bug! 

One comment - can you rename `show_st_result.sh` to `toshow_translation_result.sh` so that it works for both mt and st. Following the PR - https://github.com/espnet/espnet/pull/4146

**Comment by ftshijt** [comment]
Cool! It looks cool to me, there is one case that needs to concerned is the sacrebleu has a different layout for 1.5.1 and 2.0, maybe we should also fix the change there?

File: egs2/TEMPLATE/asr1/scripts/utils/show_st_result.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```
Comment (ftshijt) [inline egs2/TEMPLATE/asr1/scripts/utils/show_st_result.sh:1]: We can just remove the file

**Comment by sw005320** [comment]
Can you merge the latest master?
After that, I'll merge this PR.

**Comment by sw005320** [comment]
Thanks, @brianyan918!

**Pull request merged**